Altis_Life.Altis/core/shops/fn_chopShopSold.sqf

```diff
@@ -0,0 +1,19 @@
+#include "..\..\script_macros.hpp"
+/*
+	File: fn_chopShopSold.sqf
+	Author: Casperento
+	
+	Description:
+	Finish chop shop sell process properly
+*/
+params [
+	["_price",-1,[-1]],
+	"_displayName"
+];
+
+life_action_inUse = false;
+
+if (_price > 0) then {
+	CASH = CASH + _price;
+	[1,"STR_NOTF_ChopSoldCar",true,[_displayName,[_price] call life_fnc_numberText]] call life_fnc_broadcast;
```

Comment:
Maybe more elegant to just do this manually?

Reply:
Do you mean the cash operation ?

Reply:
Yeah, sure